src/os/bluestore/KernelDevice.cc

```diff
@@ -46,6 +46,7 @@ KernelDevice::KernelDevice(aio_callback_t cb, void *cbpriv)
 {
   zeros = buffer::create_page_aligned(1048576);
   zeros.zero();
+  rotational = true;
```

Comment:
and this should actually use common/blkdev.cc to figure out what is going on (via an ioctl or the /sys/block changes that are floating around)